ruby/example.rb

```diff
@@ -7,7 +7,7 @@
 # If you do you can create a context with `Extism#new`, example: `ctx = Extism.new`
 Extism.with_context do |ctx|
   manifest = {
-    :wasm => [{ :path => "../wasm/code.wasm" }],
+    :wasm => [{ :path => "./code.wasm" }],
```

Comment:
should we drop the ./ prefix so it's compatible on windows? if it's at this path I think it's ok to omit